magento/magento2
# Update varnish configuration with additional marketing tags

### Description
This PR adds additional marketing tags to the Varnish configuration so that the cache hit rate for Varnish will increase creating a better user experience. 

### Fixed Issues
1. Fixes magento/magento2#38298

### Manual testing scenarios
1. Export the varnish configuration in the admin to see that the additional parameters are there and are being excluded. 

### Contribution checklist
